_includes/header.html

```diff
@@ -7,14 +7,14 @@
           {% if page.permalink != '/' -%}
           <a class="navbar-brand title font-weight-lighter" href="{{ site.baseurl }}/">
             {%- if site.title == "blank" -%}
-              {%- if site.first_name -%}
-                <span class="font-weight-bold">{% t main.first_name %}&nbsp;</span>
+              {% if site.translations[site.lang].main.first_name %}
+                <span class="font-weight-bold">{{ site.translations[site.lang].main.first_name }}</span>
               {%- endif -%}
-              {%- if site.middle_name -%}
-                {% t main.middle_name %}&nbsp;
```

Comment:
Just out of curiosity, you removed the `&nbsp;` part. What does it do?

Reply:
It's supposed to prevent a breaking of lines, but that's already prevented even without that character. So it was just superfluous, so I didn't see a need to include it in the new code. See this example with the new code in full screen:
![Screenshot_2023-07-21_17-34-53](https://github.com/george-gca/multi-language-al-folio/assets/96623786/d7ad72cd-9b7d-4bc4-b606-1b00a42bb56f)
And in mobile screen:
![Screenshot_2023-07-21_17-35-11](https://github.com/george-gca/multi-language-al-folio/assets/96623786/447ef6e5-f744-47f0-933b-0605365ad187)

The actual reason for removing it was that it introduced double spaces in some scenarios for me with the new code and the elimination of superfluous items was easier than fiddling around to make it accommodate the superfluous item.

Reply:
I altered the code, so it's explicit with &nbsp; with cd3fa1e again (just to keep it more similar to the original; like I said, it doesn't change the function).